public/components/other.react.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default class ReactApp extends React.Component {
```

Comment:
This should probably be `class OtherComponent` It'll work without renaming, but for debugging it'll surface this name, so nice to keep them unique.